docs/guide/acquiring_data/fido.rst

```diff
@@ -26,42 +26,55 @@ properties using SunPy's attrs module::
+    Results from 1 Provider:
 
-Queries can be made more flexible or specific by adding more attrs
-objects to the Fido search.  Specific passbands taken with given
-instruments can be searched for by supplying an astropy Quantity to
-the Wavelength attribute::
+    3 Results from the LYRAClient:
+        Start Time           End Time      Source Instrument Wavelength
+          str19               str19         str6     str4       str3
+    ------------------- ------------------- ------ ---------- ----------
+    2012-03-04 00:00:00 2012-03-06 00:00:00 Proba2       lyra        nan
+    2012-03-04 00:00:00 2012-03-06 00:00:00 Proba2       lyra        nan
+    2012-03-04 00:00:00 2012-03-06 00:00:00 Proba2       lyra        nan
+
+Queries can be made more flexible or specific by adding more attrs objects to
+the Fido search. Specific passbands taken with given instruments can be searched
+for by supplying an `~astropy.units.Quantity` to the Wavelength attribute::
 
     >>> import astropy.units as u
-    >>> result = Fido.search(a.Time('2012/3/4', '2012/3/6'), a.Instrument('norh'), a.Wavelength(17*u.GHz))
+    >>> result = Fido.search(a.Time('2012/3/4', '2012/3/6'), a.Instrument('norh'),
+                             a.Wavelength(17*u.GHz))
 
 Data of a given cadence can also be specified using the Sample attribute. As
 `~sunpy.net.attrs.vso.Sample` is only supported by the `sunpy.net.vso.VSOClient`
 you specify sample with ``a.vso.Sample``. Attributes like this which are client
 specific will result in ``Fido`` only search that client, in this case VSO.::
 
-    >>> result = Fido.search(a.Time('2012/3/4', '2012/3/6'), a.Instrument('aia'), a.Wavelength(171*u.angstrom), a.Sample(10*u.minute))
+    >>> result = Fido.search(a.Time('2012/3/4', '2012/3/6'), a.Instrument('aia'),
+                             a.Wavelength(171*u.angstrom), a.vso.Sample(10*u.minute))
+
+To search for data from multiple instruments, wavelengths, times etc., use the
+pipe ``|`` operator. This joins queries together just as the logical ``OR``
+operator would::
 
-To search for data from multiple instruments, wavelengths, times etc.,
-use the pipe ``|`` operator.  This joins queries together just as the
-logical ``OR`` operator would::
+    >>> result = Fido.search(a.Time('2012/3/4', '2012/3/6'),
+                             a.Instrument('lyra') | a.Instrument('rhessi'))
 
-    >>> result = Fido.search(a.Time('2012/3/4', '2012/3/6'), a.Instrument('lyra') | a.Instrument('rhessi'))
+    >>> result = Fido.search(a.Time('2012/3/4', '2012/3/6'), a.Instrument('aia'),
+                             a.Wavelength(171*u.angstrom) | a.Wavelength(94*u.angstrom))
 
-    >>> result = Fido.search(a.Time('2012/3/4', '2012/3/6'), a.Instrument('aia'), a.Wavelength(171*u.angstrom) | a.Wavelength(94*u.angstrom))
+The above query searches the Virtual Solar Observatory (VSO) showing that Fido
+can be used instead of explicitly searching the VSO.
```

Comment:
I feel like this could be phrased slightly more clearly. Do you mean that the fact that Fido gives you results from the VSO demonstrates that you don't need to search the VSO directly yourself?